source/_integrations/songpal.markdown

```diff
@@ -30,12 +30,21 @@ A few notes:
 
 Supported devices will be automatically discovered and can be set up through UI. If you want to manually configure it in YAML configuration, add the following to your `configuration.yaml` file:
 
+for soundbars and receivers
 ```yaml
 songpal:
   - name: my soundbar
     endpoint: http://IP_ADDRESS:10000/sony
 ```
 
+for wireless speakers:
+```yaml
+songpal:
+  - name: my speakers
+    endpoint: http://IP_ADDRESS:54480/sony
+```
```

Comment:
The different ports should be mentioned in the description of the `endpoint`.

`python-songpal`s discovery feature is providing that information.

Reply:
Unresolving this review comment, @YGH-Coderm, as requested, please add the ports thing in the `endpoint` option description as well.